easybuild/easyconfigs/r/revbayes/revbayes-1.0.11-foss-2018b.eb

```diff
@@ -0,0 +1,40 @@
+#
+easyblock = 'MakeCp'
+
+name = 'revbayes'
```

Comment:
@fizwit Please rename to `RevBayes`, which seems to be the official software name based on the description?